extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+|**default**|`boolean,number,string,array`|A default value to use when the property value is not defined. If used, `optional` must be set to true. The type of the default value must match the property definition: For `BOOLEAN` use `true` or `false`. For `STRING` use a JSON string. For numeric types use a JSON number. For `ENUM` use the enum `name`, not the integer value. For `ARRAY` use a JSON array containing values matching the `componentType`.|No|
+|**optional**|`boolean`|If true, this property is optional.|No, default: `false`|
+|**semantic**|`string`|An identifier that describes how this property should be interpreted. The semantic cannot be used by other properties in the class.|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+<!-- omit in toc -->
+#### class.property.name
+
+The name of the property, e.g. for display purposes.
+
+* **Type**: `string`
+* **Required**: No
+* **Minimum Length**`: >= 1`
+
+<!-- omit in toc -->
+#### class.property.description
+
+The description of the property.
+
+* **Type**: `string`
+* **Required**: No
+* **Minimum Length**`: >= 1`
+
+<!-- omit in toc -->
+#### class.property.type
+
+The property type. If `ENUM` is used, then `enumType` must also be specified. If `ARRAY` is used, then `componentType` must also be specified. `ARRAY` is a fixed-length array when `componentCount` is defined, and variable-length otherwise.
+
+* **Type**: `string`
+* **Required**:  &#10003; Yes
+* **Allowed values**:
+   * `"INT8"`
+   * `"UINT8"`
+   * `"INT16"`
+   * `"UINT16"`
+   * `"INT32"`
+   * `"UINT32"`
+   * `"INT64"`
+   * `"UINT64"`
+   * `"FLOAT32"`
+   * `"FLOAT64"`
+   * `"BOOLEAN"`
+   * `"STRING"`
+   * `"ENUM"`
+   * `"ARRAY"`
+
+<!-- omit in toc -->
+#### class.property.enumType
+
+An enum ID as declared in the `enums` dictionary. This value must be specified when `type` or `componentType` is `ENUM`.
+
+* **Type**: `string`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.property.componentType
+
+When `type` is `ARRAY` this indicates the type of each component of the array. If `ENUM` is used, then `enumType` must also be specified.
+
+* **Type**: `any`
+* **Required**: No
+* **Allowed values**:
+   * `INT8`
+   * `UINT8`
+   * `INT16`
+   * `UINT16`
+   * `INT32`
+   * `UINT32`
+   * `INT64`
+   * `UINT64`
+   * `FLOAT32`
+   * `FLOAT64`
+   * `BOOLEAN`
+   * `STRING`
+   * `ENUM`
+
+<!-- omit in toc -->
+#### class.property.componentCount
+
+The number of components per element for `ARRAY` elements.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 2`
+
+<!-- omit in toc -->
```

Comment:
Some readers will read the spec top down / browse for learning.

Other readers will come directly to the reference for a specific piece of information.

The reference is sometimes meaty and other times extremely light.

It would be nice for the reference to link to the corresponding parts of the **Concepts** section for more context but I know that is a bit painful at the moment with the wetzel workflow - so maybe write an issue to do this in the future?